### ISSUE TITLE
acquireTokenByClientCredential is returning an invalid token

### ISSUE THREAD

**Issue by TiagoNH** [AzureAD/microsoft-authentication-library-for-js#5051]
### Core Library

MSAL Node (@azure/msal-node)

### Core Library Version

1.12.0

### Wrapper Library

Not Applicable

### Wrapper Library Version

None

### Description

I'm trying to retrieve a valid access token, but it happens that the token I'm getting is not valid, I mean, if I put it on jwt.ms website it cannot be decoded.
At first glance the string looks like a valid jwt token, but when I look closely I find the character "…" somewhere in the middle, which is odd.
I tried to do a POST call manually with the same info, and the access token in the response is in fact valid.
So can there be a bug on the msal-node ConfidentialClientApplication.acquireTokenByClientCredential method?

### Msal Logs

@azure/msal-node@1.12.0 : Verbose - buildOauthClientConfiguration called
[Fri, 29 Jul 2022 13:23:46 GMT] : [3f82d7d9-aab5-456c-8517-294a4a5989c1] : @azure/msal-node@1.12.0 : Verbose - building oauth client configuration with the authority: https://login.microsoftonline.com/tenant-id
[Fri, 29 Jul 2022 13:23:46 GMT] : [3f82d7d9-aab5-456c-8517-294a4a5989c1] : @azure/msal-node@1.12.0 : Verbose - createAuthority called
[Fri, 29 Jul 2022 13:23:46 GMT] : [3f82d7d9-aab5-456c-8517-294a4a5989c1] : @azure/msal-node@1.12.0 : Verbose - Client credential client created
[Fri, 29 Jul 2022 13:23:47 GMT] : [3f82d7d9-aab5-456c-8517-294a4a5989c1] : @azure/msal-common@7.2.0 : Verbose - No client info in response

### MSAL Configuration

```javascript
const config = {
  auth: {
    clientId: 'clientID',
    authority: 'https://login.microsoftonline.com/tenant-id',
    clientSecret: 'clientSecret',
  },
  system: {
    loggerOptions: {
      loggerCallback(loglevel, message, containsPii) {
        console.log(message);
      },
      piiLoggingEnabled: false,
      logLevel: LogLevel.Verbose,
    },
  },
}
```

### Relevant Code Snippets

```javascript
const cca = new ConfidentialClientApplication(config);
const ccr: ClientCredentialRequest = {
          scopes: ['https://[tenant].onmicrosoft.com/web-api/.default'],
        };
        const result = await cca.acquireTokenByClientCredential(ccr);
        console.log(result.accessToken);
```

### Reproduction Steps

Running the code and checking the returned access token

### Expected Behavior

A valid jwt access token instead of an invalid one

### Identity Provider

Azure AD / MSA

### Browsers Affected (Select all that apply)

None (Server)

### Source

External (Customer)

**Comment by TiagoNH** [comment]
sorry, found the issue

**Comment by danljs** [comment]
@TiagoNH Encountered a similar problem, how did you solve it?

**Comment by TiagoNH** [comment]
@danljs there were no issue at all, it was my mistake...I was checking the token through the debugger, which apparently doesn't show the entire string. So, when I picked the token with a console.log and checked it, it was OK.

And I really thought my token was invalid, because when I was using it to authenticate on the web API, I was getting an authentication error. But now I found the real reason for why the token is not working, and  it doesn't have anything to do with the msal library.
